homeassistant/components/sma/sensor.py

```diff
@@ -134,6 +135,7 @@ async def async_setup_entry(
     coordinator = sma_data[PYSMA_COORDINATOR]
     used_sensors = sma_data[PYSMA_SENSORS]
     device_info = sma_data[PYSMA_DEVICE_INFO]
+    device_info["configuration_url"] = sma_data[PYSMA_OBJECT].url
```

Comment:
Import constant from core `const.py`: https://github.com/home-assistant/core/blob/9daf2ee65d617055b7b17783a369e4b5f5fe4006/homeassistant/const.py#L355

```suggestion
    device_info[ATTR_CONFIGURATION_URL] = sma_data[PYSMA_OBJECT].url
```